src/components/specimen/Specimen.tsx

```diff
@@ -203,51 +204,56 @@ const Specimen = () => {
-                                <Col className={`${classSpecimenContent} h-100 transition`}>
-                                    <div className="h-100 d-flex flex-column">
-                                        <Row className="titleBar">
-                                            <Col>
-                                                <TitleBar ShowWithAllAnnotations={() => ShowWithAnnotations()}
-                                                    ToggleAutomatedAnnotations={() => setAutomatedAnnotationToggle(!automatedAnnotationsToggle)}
-                                                />
-                                            </Col>
-                                        </Row>
-                                        <Row className="py-4 flex-grow-1 overflow-scroll overflow-lg-hidden">
-                                            <Col lg={{ span: 3 }} className={`${classIdCard} IDCard`}>
-                                                <IDCard />
-                                            </Col>
-                                            <Col lg={{ span: 9 }} className="contentBlock ps-4 h-100 mt-4 m-lg-0">
-                                                <ContentBlock selectedTab={selectedTab}
-                                                    SetSelectedTab={(tabIndex: number) => setSelectedTab(tabIndex)}
-                                                />
-                                            </Col>
-                                        </Row>
-                                    </div>
-                                </Col>
-                            </Row>
-                        </Container>
-                    }
-
-                    <Footer />
+                    <div className="h-100 d-flex flex-column">
+                        <Header introTopics={[
+                            { intro: 'specimen', title: 'About This Page' },
+                            { intro: 'annotate', title: 'Using Annotations' },
+                            { intro: 'MAS', title: 'Machine Annotation Services' }
+                        ]} />
+
+                        <SpecimenSteps SetSelectedTab={(tabIndex: number) => setSelectedTab(tabIndex)} />
+                        <AnnotateSteps ShowWithAnnotations={(annotations?: SpecimenAnnotations, property?: string) => ShowWithAnnotations(annotations, property)} />
+                        <MASSteps automatedAnnotationsToggle={automatedAnnotationsToggle}
+                            SetAutomatedAnnotationsToggle={(toggle: boolean) => setAutomatedAnnotationToggle(toggle)}
+                            ShowWithAnnotations={() => ShowWithAnnotations()}
+                        />
+
+                        <div className="flex-grow-1 overflow-hidden">
+                            {(specimen.digitalSpecimen['ods:id'] &&
+                                specimen.digitalSpecimen['ods:id'].replace('https://doi.org/', '') === `${params['prefix']}/${params['suffix']}`) &&
```

Comment:
Looks like a white line that doesn't need to be there

Reply:
Did this on purpose because the line got too long in the editor, but reverted this to one line